GithubRepository/src/repositoryUtils.js

```diff
@@ -0,0 +1,21 @@
+
+// Function to filter the repositiries with more than 5 stars
+function filterRepositoriesWithMoreThanFiveStars(repositories) {
+    return repositories.filter(repo => repo.stargazers_count > 5);
+}
+
+
+function getLastFiveUpdatedRepositories(repositories) {
+    //Sort the repositories desc by date
+    const sortedRepositories = repositories.sort((a, b) => new Date(b.updated_at) - new Date(a.updated_at));
+    // Ttake de last 5 repositories
+    return sortedRepositories.slice(0, 5);
+}
```

Comment:
Commenting code is OK, but it is better if the code is readable enough, in this case, it is useless to comment on every step.